### ISSUE TITLE
W605: identical tokens cause infinite loop 

### ISSUE THREAD

**Issue by bigredengineer** [hhatto/autopep8#449]
There is a bug with the way that tokens are handled with W605 that causes an infinite loop to occur when two identical W605 tokens exist on the same line of code.  This infinite loop can be observed both on the command line and by adding a test to the test_autopep8.py unit test file.

**Test Case Code:**
`escape = foo('\.bar', '\.bar')\n`

**Command Line:**
`autopep8.py src/ -i -r --exclude "config_*" -a -j 8`

**Workaround:** 
- To the command line, add `--pep8-passes <n>`.  This will break the infinite loop, but a bad code-fix is put in place
- Bad Code-Fix Example =  `escape = foo(rrrrrrrrr'\.bar', '\.bar')\n`

**Version Info**
`autopep8.py --version  :  autopep8 1.4.1 (pycodestyle: 2.4.0)`
`/usr/bin/env python --version  : Python 2.7.5`

**Comment by hhatto** [comment]
planning to release version 1.4.3.

**Comment by hhatto** [comment]
version 1.4.3 released
https://pypi.org/project/autopep8/1.4.3/

**Comment by stmlange** [comment]
This still seems to be an issue again with 1.4.4:
```
$ python -m autopep8 --version
autopep8 1.4.4 (pycodestyle: 2.5.0)

$ cat << EOF > foo.py                                                   
import re
re.findall(rr'([0-9]+)([^\s]+)', "122122aaaaaa")
EOF

$ python -m autopep8 -i --ignore errors=E401,E402 -aaa --verbose foo.py
```

**Comment by hhatto** [comment]
fix in #514 .

planning to release version 1.5.

Thanks
